src/content/tooling/wrangler/commands.md

```diff
@@ -138,6 +138,17 @@ If you would like to be able to publish your code to multiple places, please see
   If you have feedback about `wrangler dev` or general questions, we will respond [here](https://github.com/cloudflare/wrangler/issues/1047).
 
 
+### tail
```

Comment:
Should show the  syntax  and options as in `init`
```
wrangler tail  --port PORT --metrics-port PORT
```
Or similar to show flags as optional

Reply:
A concise definition of the command should live here at the top under the syntax

Reply:
+1 to adding a usage example (as long as we state that the flags are optional)

Reply:
added; used the bracket notation for indicating these were optional.